bin/versions.sh

```diff
@@ -39,8 +39,7 @@ fi
 
 if [[ $LANGUAGES == *":lua:"* ]]; then
     echo -n "lua,"
-    # scilua -v 2>&1 | grep Shell | cut -f3 -d" " | cut -f1 -d,
-    echo scilua v1.0.0-b12
+    (cd lua; ./ulua/luajit/2_1_head20151128/Linux/x64/luajit -v | cut -f2 -d" ")
```

Comment:
Is `2_1_head20151128` specific to the latest version? If it is, I would swap it with a `*`, for future versions.

Reply:
That's a great idea!